.github/workflows/ci.yml

```diff
@@ -534,6 +534,16 @@ jobs:
                 github.event.client_payload.pull_request.head.sha == github.event.client_payload.slash_command.args.named.sha &&
                 format('refs/pull/{0}/head', github.event.client_payload.pull_request.number) || '' }}
       - uses: ./.github/actions/setup
+      - name: Update PR check
+        uses: ./.github/actions/update-check
+        if: >-
+          always() &&
+          github.event_name == 'repository_dispatch' &&
+          github.event.client_payload.slash_command.args.named.sha != '' &&
+          github.event.client_payload.pull_request.head.sha == github.event.client_payload.slash_command.args.named.sha
+        with:
+          pull_request_number: ${{ github.event.client_payload.pull_request.number }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this is unrelated right? Because we already invoke this from the `test` job.

Reply:
We need to invoke this action twice, to add the comment when the workflow starts (so as early as possible) and when it finishes (fails).